tools/login/actions/login.php

```diff
@@ -125,8 +147,13 @@
         if ($existingUser["password"] === md5($_POST["password"])) {
             $this->SetUser($existingUser, $_POST["remember"]);
 
+            // si l'on veut utiliser l'url spécifié par $loggedinurl
+            if (isset($loggedinurl))
+            {
+            	$this->Redirect($this->href('', $loggedinurl, ''));
+            }
```

Comment:
```suggestion
            } else {
              $this->Redirect($loggedinurl);
            }
           }
```
pour être plus table si ça n'est pas une page bazar

Reply:
Possiblement. Encore une fois j'ai eu du mal à comprendre les parametres de cette fonction Redirect